packages/nollup/package.json

```diff
@@ -4,7 +4,8 @@
 	"main": "src/index.js",
 	"exports": {
```

Comment:
Mind adding it here as `"module": "src/wrapper.js'` as well?

Reply:
Added in https://github.com/JoviDeCroock/prefresh/pull/223/commits/fd1bed8ea58c9a0fda8f79750b15fc3bc505ae2f